docs/resources/vdi.md

```diff
@@ -0,0 +1,34 @@
+# xenorchestra_vdi
+
+Creates a Xen Orchestra cloud config resource.
+
+## Example Usage
+
+```hcl
+resource "xenorchestra_vdi" "bar" {
+    name_label = "alpine-virt-3-17-0"
+    sr_id = data.xenorchestra_sr.sr.id
+    filepath = "${path.module}/isos/alpine-virt-3.17.0-x86_64.iso"
+    type = "raw"
+}
+
+# Use the vdi with the VM resource
+resource "xenorchestra_vm" "bar" {
+
+  # Other required options omitted
+  # [ ... ]
+
+  cdrom {
+    id = resource.xenorchestra_vdi.bar.id
+  }
+}
+```
+
+## Argument Reference
+* name_label - (Required) The name label of the VDI
+* sr_id - (Required) The id of the Storage repository the VDI should be created in.
```

Comment:
We should probably emphasize that while it is possible to import an ISO into a normal sr, it most likely isn't useful.
This goes the other way around as well.

Reply:
Good call. I figured that was the case, but I don't have any practical experience with it.